### PR TITLE
Small edits to NFS page

### PR DESCRIPTION
**Pull request by sascala**
@jgarcia-mesosphere If these look good to you, please reassign to Joel.

### CODE REVIEW

File: 1.7/administration/storage/nfs.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ menu_order: 1
...
 
-This example uses CoreOS and systemd and has not been tested in other environments.
+**Note:** This example uses CoreOS and systemd and has not been tested in other environments.
 
 ## Notes
 
-- These instructions are CoreOS specific.
-- This is not an HA NFS server. It is being run on a single master and should only be used as a proof of concept.
+- These instructions are CoreOS-specific.
+- This is not an HA NFS server. It runs on a single master and should only be used as a proof of concept.
```
Comment (jgarcia-mesosphere) [inline 1.7/administration/storage/nfs.md]: Would also like to add that most operations teams will have access to NFS or another network store, and those folks can skip right to Part 2.

Comment (sascala) [inline 1.7/administration/storage/nfs.md]: Done

Comment (sascala) [inline 1.7/administration/storage/nfs.md]: Done.

**Comment by emanic** [comment]
👍

**Pull request merged**